asciidoc-confluence-publisher-client/src/test/java/org/sahli/asciidoc/confluence/publisher/client/ConfluencePublisherTest.java

```diff
@@ -239,8 +240,8 @@ public void publish_metadataOnePageWithNewAttachmentsAndAncestorIdAsRoot_attache
         assertThat(contentId.getAllValues(), contains("4321", "4321"));
         assertThat(inputStreamAsString(attachmentContent.getAllValues().get(attachmentFileName.getAllValues().indexOf("attachmentOne.txt")), UTF_8), is("attachment1"));
         assertThat(inputStreamAsString(attachmentContent.getAllValues().get(attachmentFileName.getAllValues().indexOf("attachmentTwo.txt")), UTF_8), is("attachment2"));
-        verify(confluenceRestClientMock).setPropertyByKey("4321", "attachmentOne.txt-hash", sha256Hex("attachment1"));
-        verify(confluenceRestClientMock).setPropertyByKey("4321", "attachmentTwo.txt-hash", sha256Hex("attachment2"));
+        verify(confluenceRestClientMock).setPropertyByKey("4321", getAttachmentHashKey("attachmentOne.txt"), sha256Hex("attachment1"));
```

Comment:
Instead of using the same method for calculating the attachment has key both in the test and in the production code, I would prefer to use the expected literal in the test (e.g. `attachmentOne-attachment-hash`), without calling the production code. Currently, the test does not communicate the expectation very well, plus there is the risk of a "self-fulfilling prophecy" (the test can be green even though the implementation is buggy). 

That way, there should also be no need anymore to make the method `getAttachmentHasKey` protected and static.